nucleus/validate/scenario_test.py

```diff
@@ -178,6 +185,11 @@ def get_items(
         response = self.connection.get(
             f"validate/scenario_test/{self.id}/items",
         )
+        if level == EntityLevel.TRACK:
+            return [
+                Track.from_json(track, connection=self.connection)
```

Comment:
This line is the reason for the long justification above 😤

Reply:
Haha, the struggle is real 🥇